sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateOperationTests.cs

```diff
@@ -288,23 +288,7 @@ private CertificateClient CreateClient(HttpPipelineTransport transport)
 
         private async ValueTask<KeyVaultCertificateWithPolicy> WaitForOperationAsync(CertificateOperation operation)
         {
-            var rand = new Random();
-            TimeSpan PollingInterval() => TimeSpan.FromMilliseconds(rand.Next(1, 50));
-
-            if (IsAsync)
-            {
-                return await operation.WaitForCompletionAsync(PollingInterval(), default);
-            }
-            else
-            {
-                while (!operation.HasCompleted)
-                {
-                    operation.UpdateStatus();
-                    await Task.Delay(PollingInterval());
-                }
-
-                return operation.Value;
-            }
+            return await operation.WaitForCompletionAsync();
```

Comment:
Off hand, I can't remember why I did this, but `CertificateOperation` tests have been nortoriously flaky and this would've been to help somehow.

While I'm not *super* opposed to making this change, I am concerned that passing in a different polling interval doesn't work with your changes. Or does it (I thought the interceptor checked the args), and you just thought you'd replace this anyway?

Reply:
>Or does it (I thought the interceptor checked the args), and you just thought you'd replace this anyway?

It does, in Live/Recorded tests the interceptor no-ops.